### PR TITLE
please work

### PR DESCRIPTION
**Pull request by hakosx**
# Changes :gem:
merge to master
🙏 🙏

### CODE REVIEW

File: src/webapp/models.py [hunk 1]
```diff
@@ -17,6 +17,7 @@ class Iteration(db.Model):
     """
         Model for storing the currently used iteration of the ML model.
     """
+
```
Comment (andreacx) [inline src/webapp/models.py:20]: thought we decided to not have new line here?

File: src/webapp/models.py [hunk 2]
```diff
@@ -26,6 +27,7 @@ class Games(db.Model):
        inserted values match the column values. Token column value cannot
        be String when a long hex is given.
     """
+
```
Comment (joaaax) [inline src/webapp/models.py:30]: Newline isn't necessary

File: src/webapp/models.py [hunk 3]
```diff
@@ -37,6 +39,7 @@ class Scores(db.Model):
         This is the Scores model in the database. It is important that the
         inserted values match the column values.
     """
+
```
Comment (joaaax) [inline src/webapp/models.py:42]: Neither here

File: src/webapp/models.py [hunk 4]
```diff
@@ -48,9 +51,10 @@ class PlayerInGame(db.Model):
         Table for attributes connected to a player in the game. game_id is a
         foreign key to the game table.
     """
+
```
Comment (joaaax) [inline src/webapp/models.py:54]: Or here

File: src/webapp/models.py [hunk 5]
```diff
@@ -60,6 +64,7 @@ class Labels(db.Model):
         - translating english labels into norwgian
         - keeping track of all possible labels
     """
+
```
Comment (joaaax) [inline src/webapp/models.py:67]: Or here

**Pull request merged**